src/modules/mana/utils.ts

```diff
@@ -1,2 +1,4 @@
 export const MANA_CONTRACT_ADDRESS = process.env
   .REACT_APP_MANA_CONTRACT_ADDRESS!
+export const ERC20_PREDICATE = process.env.REACT_APP_ERC20_PREDICATE!
+export const ROOT_CHAIN_MANAGER = process.env.REACT_APP_ROOT_CHAIN_MANAGER!
```

Comment:
lets add `_CONTRACT_ADDRESS` to the end of these 2  constants, that's usually the namig convention we use across projects for contract addresses, ie: `ERC20_PREDICATE_CONTRACT_ADDRESS` and `ROOT_CHAIN_MANAGER_CONTRACT_ADDRESS`

Reply:
Agree with you! fixing!